README.md

```diff
@@ -27,7 +27,11 @@ Use Quix Streams to build simple Kafka producer/consumer applications or leverag
 ### Install Quix Streams
 
 ```shell
+# PyPI
 python -m pip install quixstreams
```

Comment:
hey, lets use this opportunity to simplify it to `pip install quixstreams`?

Reply:
There was a [discussion](https://bugs.python.org/issue22295) about it too.
I don't remember all the differences, but `python -m pip` seems to be more compatible with different OS and ensures the package is installed to a correct interpreter